integration/tests/cook/test_cli.py

```diff
@@ -963,6 +963,18 @@ def test_submit_with_executor(self):
         self.assertEqual(2, cp.returncode, cp.stderr)
         self.assertIn("invalid choice: 'bogus'", cli.decode(cp.stderr))
 
+    def test_job_environment(self):
+        """Note: This test doesn't directly test the cli. It relies on stdout to test the job environment."""
```

Comment:
It's a little odd that this lives in `test_cli` but isn't meant to test the CLI

Reply:
I know :( 
Any suggestions on how to better test the feature?

Reply:
Can we fetch job output without using `cs tail`?

Reply:
None of the other non-cli tests do it and figured it would pretty much repeat what `cs tail` does.
I considered progress messages, but didn't want to tie this test to the cook executor.

Reply:
Even though none of the other tests do it currently, it does seem like a generally useful util function for tests to get the stdout, stderr from a job.

As for the duplication, we could consider making the util function just shell out to `cs tail`

Reply:
I think I'll rewrite the command to check and env and exit `1` if it doesn't find those entries in the environment. That way we can avoid the dependency on the cli.

Reply:
Rewrote the test.